packages/shipjs/src/step/release/createGitHubRelease.js

```diff
@@ -0,0 +1,71 @@
+import fs from 'fs';
+import path from 'path';
+import tempWrite from 'temp-write';
+
+import runStep from '../runStep';
+
+function getChangelog(version) {
+  const changelogMatcher = new RegExp(
+    `#+?[\\s\\[]*?(${version})(.|\n)+?(?=#+?[\\s\\[]*?\\d\\.\\d|$)`
+  );
+  const changelogPath = fs.resolve(process.cwd(), 'CHANGELOG.md');
+  try {
+    const changelogFile = fs.readFileSync(changelogPath, 'utf-8').toString();
+    const changelogMatch = changelogFile.match(changelogMatcher);
+    if (changelogMatch !== null) {
+      return changelogMatch[0];
+    }
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      return null;
+    }
+    throw err;
+  }
+  return null;
+}
+
+export default ({ version, config, dir, dryRun }) =>
+  runStep({ title: 'Creating the GitHub Release' }, ({ run }) => {
+    if (config.updateChangelog !== true) return;
+
+    const { getTagName } = config;
+    const tagName = getTagName({ version });
+
+    const existingChangelog = getChangelog(version);
+    let exportedPath;
+    if (existingChangelog !== null) {
+      // fetch CHANGELOG
+      exportedPath = tempWrite.sync(existingChangelog);
+    } else {
+      // generate CHANGELOG
```

Comment:
What do you think of not generating here at all?
For example, at `shipjs prepare`, conventional-changelog already generated some changelog for the current version and user didn't like it, so they removed it. And if we generate and use it again for the github release, there's nothing user can do about it.
It might make sense just to keep using what user decided at `shipjs prepare` step.
What do you think?

Reply:
I initially thought it was good to generate CHANGELOG automatically for the release but then realized that it was supposed to be already generated before `trigger`. So you are right.
Let this section just extract matching release from CHANGELOG then generate a release on GitHub repository.

Reply:
> How about just create a release with a blank message if there are no CHANGELOG?

It totally makes sense 👍

Reply:
@eunjae-lee 👌👌
I already updated my code to do so

Reply:
Nice! I'll try out this PR this weekend, and hopefully merge it 🙂

Reply:
@eunjae-lee Yay! Thank you for taking your weekend time ☺️